doc/manual/src/command-ref/nix-instantiate.md

```diff
@@ -44,9 +44,10 @@ standard input.
 
     > **Warning**
     >
-    > This option produces ambiguous output which is not suitable for machine
-    > consumption. For example, these two Nix expressions print the same result
-    > despite having different types:
+    > This option produces output which can be parsed as a Nix expression which
+    > will produce a different result than the input expression when evaluated.
+    > For example, these two Nix expressions print the same result despite
+    > having different values:
```

Comment:
```suggestion
    > having different meaning:
```

I don't think we want _value_ because that term should be quite specific and I'm not sure if it applies here. Depends on definitions, but that just makes is more complicated than the message we want to get across.